README.md

```diff
@@ -31,14 +31,17 @@ This tap:
     ```json
     {
         "start_date": "2010-01-01",
-        "api_key": "your-api-token"
+        "api_key": "your-api-token",
+        "activities_date_window": 10
```

Comment:
```suggestion
        "date_window": 10
```

Reply:
Use a generic name so in the future, if anyone wants to use it for any other stream then they can.

Reply:
Added.